### DISCUSSION TITLE
UI suggestions

### DISCUSSION THREAD

**Discussion by eggbean** [Devolutions/UniGetUI#1243] [General]
Hi, I find this useful, but..

I think this could be more useful if text was selectable (and therefore copyable) in the UI, so that notes/scripts can be easily made.

Also, in the Installed Packages list it would be good if there was a way to filter Source types, as it would make it a lot easier to find what you are looking for.

And there is a unnecessary alert sound every time the Are You Sure? dialogue box comes up when trying to uninstall something which is just irritating.

**Comment by marticliment** [comment]
> And there is a unnecessary alert sound every time the Are You Sure? dialogue box comes up when trying to uninstall something which is just irritating.

You can select multiple packages and uninstall them at once, and only one popup will be shown.



> Also, in the Installed Packages list it would be good if there was a way to filter Source types, as it would make it a lot easier to find what you are looking for.

This feature is in the priority list

> I think this could be more useful if text was selectable (and therefore copyable) in the UI, so that notes/scripts can be easily made.

Where do you mean?

**Comment by eggbean** [comment]
It is not possible to select text in the window. For example, I want to make a list of all the package IDs of packages I have installed, so that I can script the installation on new OS installations.

**Reply by ppvnf** [reply]
you can do it using this button (it generates a json file with the package information)

<img width="959" alt="Screenshot 2023-08-05 123534" src="https://github.com/marticliment/WingetUI/assets/73800734/a37ddcf5-75ae-4f13-b906-e1cfbaff874f">

and then import the json in the install section to install the packages

**Reply by eggbean** [reply]
Oh, thanks. I seem to have missed that.
